### PR TITLE
add blog post about using goal templates

### PR DESCRIPTION
**Pull request by youngcw**
We have a blog, so I wrote a post for it.

Its about how I use the goal templates in the hopes that it will inspire some people on how they can use goals and some budgeting strategy baked in.

### CODE REVIEW

File: blog/2023-12-20-automate.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+---
+title: Automate Your Budget With Goal Templates
+description: An example of how you can automate all of your budgeting by using Actual's goal template feature
+slug: 2024-01-automateyourbudget
```
Comment (MatissJanis) [inline blog/2023-12-20-automate.md:4]: **🥜 nitpick:** ‏
```suggestion
slug: 2024-01-automate-your-budget-with-goal-templates
```

File: blog/2023-12-20-automate.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+tags: [budgeting, goals]
+authors: youngcw
+hide_table_of_contents: false
+---
+
+Hello budgeters!
+I hope you are all enjoying Actual as much as I do!
+As a new year approaches so to do new goals.
```
Comment (MatissJanis) [inline blog/2023-12-20-automate.md:12]:
```suggestion
As a new year is approaching so is the goal-setting and reflection season.
```

File: blog/2023-12-20-automate.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+I'm hoping to maybe help some of you out with potential new years resolutions by making budgeting as easy as possible while still reaching for your goals.
```
Comment (MatissJanis) [inline blog/2023-12-20-automate.md:13]:
```suggestion
I'm hoping to help some of you with potential new years resolutions by making budgeting as easy as possible while still reaching for your goals.
```

File: blog/2023-12-20-automate.md [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+I believe that Actual is a great tool for this whether you have a simple budget or a complex one.
+
+One hurdle that many have with budgeting is that it takes constant work to stay up to date.
```
Comment (MatissJanis) [inline blog/2023-12-20-automate.md:16]:
```suggestion
One hurdle that many have with budgeting is that it takes constant effort to stay up to date.
```

File: blog/2023-12-20-automate.md [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+Entering transactions, assigning categories, reconciling with your bank and others can hold us back.
+Some of those are just the nature of the beast, but some can streamlined.
```
Comment (MatissJanis) [inline blog/2023-12-20-automate.md:18]: **🥜 nitpick:** ‏not sure I understand this sentence.

File: blog/2023-12-20-automate.md [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+I hope to give you some pointers and strategies today to reduce some of the stress of budgeting by streamlining the budgeting processes by using goal templates.
```
Comment (MatissJanis) [inline blog/2023-12-20-automate.md:19]:
```suggestion
I hope to give you some pointers and strategies today to reduce some of the stress of budgeting by streamlining the budgeting processes with goal templates.
```

File: blog/2023-12-20-automate.md [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+I hope that there was something in here that was useful to you in your budgeting journey.
+
+If budgeting feels daunting, don't worry.
+It is hard to get going and can be hard if things are tight, but taking the time to get on top of things now will give you more financial freedom in the future.
+
+I wish you a merry holiday season and a happy new year!
+
+Happy budgeting
+
+-- youngcw
```
Comment (MatissJanis) [inline blog/2023-12-20-automate.md:243]: **🥜 nitpick:** ‏I think you can remove this since your name is already signed at the top of the post.

**Review by MatissJanis — APPROVED** [review]
Overall LGTM! I'm not a native speaker myself, so feel free to disregard my comments. Reviewed in depth the "above-the-fold" part of the post.

**Comment by MatissJanis** [comment]
@check-spelling-bot apply [updates](https://github.com/actualbudget/docs/actions/runs/7187038797/attempts/1).

File: blog/2023-12-20-automate.md [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+* `#template 10% of all income` to budget 10% of all income categories.
+* `#template 10% of Paycheck` to only include the "Paycheck" category in the calculation.
+
+### Template Priorities
+Okay, so you understand the templates pretty well now.
+Now for, in my opinion, the most useful part of templates.
+We can set priorities!
+Setting a priority to the templates does two things for us
+1. It tells Actual to not budget more money then shows in your `To Budget`.  This is not the case by default.
+2. I tells Actual what order to apply the templates.
```
Comment (shall0pass) [inline blog/2023-12-20-automate.md:109]: It tells Actual in what (or in which) order to apply the templates.

**Review by shall0pass — COMMENTED** [review]
Other than the one grammar item, the post looks very good!  Well done!

**Review by MatissJanis — APPROVED** [review]

File: blog/2023-12-20-automate.md [hunk 9]
```diff
@@ -0,0 +1,250 @@
+---
+title: Automate Your Budget With Goal Templates
+description: An example of how you can automate all of your budgeting by using Actual's goal template feature
+slug: 2023-12-automate-your-budget-with-goal-templates
```
Comment (MatissJanis) [inline blog/2023-12-20-automate.md:4]: FYI: might want to update this date (and filename) before merging. With the latest date.

**Review by MatissJanis — APPROVED** [review]

**Pull request merged**